# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/common.js
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/tests/mochitest/common.js
--- a/accessible/tests/mochitest/common.js	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/tests/mochitest/common.js	Wed Nov 29 03:53:20 2023 +0000
@@ -24,8 +24,6 @@
 const nsIAccessibleTextChangeEvent = Ci.nsIAccessibleTextChangeEvent;
 const nsIAccessibleTextSelectionChangeEvent =
   Ci.nsIAccessibleTextSelectionChangeEvent;
-const nsIAccessibleVirtualCursorChangeEvent =
-  Ci.nsIAccessibleVirtualCursorChangeEvent;
 const nsIAccessibleObjectAttributeChangedEvent =
   Ci.nsIAccessibleObjectAttributeChangedEvent;
 const nsIAccessibleAnnouncementEvent = Ci.nsIAccessibleAnnouncementEvent;